core/test/UtGunnsFluidNode.cpp

```diff
@@ -1076,65 +1070,64 @@ void UtGunnsFluidNode::testOutflowOverflow()
+        const double expectedFractO2  = fractionsIn[0];
+        const double expectedFractN2  = fractionsIn[1];
+        const double expectedFractCO2 = fractionsIn[2];
+        const double expectedMassErr  = expectedMass - solutionMass;
 
-    tNode.integrateFlows(dt);
+        tNode.integrateFlows(dt);
 
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedMass,     tNode.getMass(),    DBL_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedNetFlux,  tNode.mNetFlux,     DBL_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFluxThru, tNode.mFluxThrough, DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedMass,     tNode.getMass(),    DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedNetFlux,  tNode.mNetFlux,     DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFluxThru, tNode.mFluxThrough, DBL_EPSILON);
 
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractO2,
-            tNode.mContent.getMassFraction(FluidProperties::GUNNS_O2),  FLT_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractN2,
-            tNode.mContent.getMassFraction(FluidProperties::GUNNS_N2),  FLT_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractCO2,
-            tNode.mContent.getMassFraction(FluidProperties::GUNNS_CO2), FLT_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractO2,
+                tNode.mContent.getMassFraction(FluidProperties::GUNNS_O2),  FLT_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractN2,
+                tNode.mContent.getMassFraction(FluidProperties::GUNNS_N2),  FLT_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedFractCO2,
+                tNode.mContent.getMassFraction(FluidProperties::GUNNS_CO2), FLT_EPSILON);
 
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(0.0,             tNode.mExpansionDeltaT,          DBL_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(290.0,           tNode.mContent.getTemperature(), FLT_EPSILON);
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedMassErr, tNode.mMassError,                DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(0.0,             tNode.mExpansionDeltaT,          DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(290.0,           tNode.mContent.getTemperature(), FLT_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(expectedMassErr, tNode.mMassError,                DBL_EPSILON);
 
-    CPPUNIT_ASSERT_DOUBLES_EQUAL(outFlow,         tNode.getOutflow()->getFlowRate(), DBL_EPSILON);
+        CPPUNIT_ASSERT_DOUBLES_EQUAL(outFlow,         tNode.getOutflow()->getFlowRate(), DBL_EPSILON);
+    }
+
+    tNode.resetFlows();
 
     /// - Now do a test with the same outflux, but the inflow is so small that it causes the node
     ///   to limit resulting mass from going negative.  The resulting mixture should equal the
     ///   inflow.
     {
-        tNode.resetFlows();
-        tNode.scheduleOutflux(thruFlux);
-        CPPUNIT_ASSERT(tNode.isOverflowing(dt));
-        const double inFlowRate = thruFlux * tFluidIn.getMWeight() * 1.0e-6;
-
```

Comment:
The move of these 4 lines to down below is unwarranted, changes what is being tested and the test loses value.  I see the need to redefine thruFlux since the previous definition went out of scope.  But these 4 lines should remain here, and then ```tNode.resetFlows();``` is redundant with above so remove one of them.  Please restore this and resubmit.

Reply:
Fixed.